bucket/winauth.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "3.6.2",
+    "description": "Portable and open-source authenticator.",
+    "homepage": "https://winauth.github.io/winauth/",
```

Comment:
```suggestion
    "homepage": "https://winauth.github.io/winauth",
```

Reply:
Is there a reason for this? The slash is automatically added anyway when the site is accessed.

Reply:
That is how browsers are handling it.

But forget this change. I forgot that github.io need slash.

![image](https://user-images.githubusercontent.com/13260377/67133115-be226d00-f20b-11e9-9582-f7dff8b559ee.png)